docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -557,33 +557,7 @@ The connector supports the following [](/sql/alter-table) statements.
 
 ##### ALTER TABLE EXECUTE
 
```

Comment:
We should add the same intro here as we have in iceberg ..

Reply:
We removed it from Delta Lake because it didn't flow with the new ALTER TABLE heading.